src/AddTodoForm.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+function AddTodoForm ({onAddTodo}){
+    const [todoTitle, setTodoTitle ] = useState ("") 
+    const handleTitleChange = (event)=> {
+        setTodoTitle(event.target.value) 
+    }
+    const handleAddTodo = (event) => {
+
+        event.preventDefault();
+        const todoTitle = event.target.title.value; 
+        console.log(todoTitle);
+        event.target.title.value = "";
```

Comment:
Nice job here!!

Your input is a controlled element now. You can achieve blanking the field by setting `todoTitle` to blank/empty now. 

Try and fix this during the next exercise 👍